.mailmap

```diff
@@ -0,0 +1,16 @@
+Tal Yarkoni <tyarkoni@gmail.com> tyarkoni <tyarkoni@gmail.com>
+Alejandro de la Vega <delavega@utexas.edu> adelavega <delavega@utexas.edu>
+Alejandro de la Vega <delavega@utexas.edu> delavega4 <aleph4@gmail.com>
+Alejandro de la Vega <delavega@utexas.edu> Alejandro de la Vega <aleph4@gmail.com>
+Christopher J. Markiewicz <markiewicz@stanford.edu> Chris Markiewicz <effigies@gmail.com>
+Christopher J. Markiewicz <markiewicz@stanford.edu> Chris Markiewicz <markiewicz@stanford.edu>
```

Comment:
Please use this line as an example for declaring how you would like your name to appear in citations.

Reply:
just to clarify, if my commit-name coincides with my true name, I don't need to add myself here, right?

Reply:
Right. Unless you want to add a middle initial or anything, you don't need to do anything here.